.circleci/config.yml

```diff
@@ -36,10 +36,11 @@ jobs:
       - run: sudo apt-get update
       - run: sudo apt-get install tree
       - run: sudo apt-get install cf-cli
+      - run: cf install-plugin https://github.com/govau/autopilot/releases/download/0.0.5-venapp/autopilot-linux -f
```

Comment:
This will work, but most Circle files I've seen have a "pre" step that installs `cf` etc - I don't really understand the difference, but just mention it in passing. Probably not work changing here...